app/src/main/java/ch/epfl/polychef/fragments/RateRecipeFragment.java

```diff
@@ -146,7 +137,7 @@ public void onAttach(@NonNull Context context) {
 
         if(context instanceof HomePage){
             HomePage homePage = (HomePage) context;
-            fireDatabase = homePage.getFireDatabase();
+            FirebaseDatabase fireDatabase = homePage.getFireDatabase();
```

Comment:
Since firebaseDatabase is local to this method, we can just remove it

Reply:
👍